src/app/auth/components/admin-panel/admin-panel.component.html

```diff
@@ -1 +1,46 @@
-<p>admin-panel works!</p>
+<div class="admin-panel-container">
+  <div class="card-creation-container">
+    <form [formGroup]="cardCreationForm" class="card-creation-form">
+      <mat-form-field appearance="fill" class="input-container">
+        <mat-label>Title</mat-label>
+        <input matInput type="text" name="title" formControlName="title">
+        <mat-error *ngIf="cardCreationForm.get('title')?.errors?.['required']">Please enter a title</mat-error>
```

Comment:
You should make this into a helper function, as you are repeating this condition more than once